packages/searchkit-schema/tests/HitsResolver.test.ts

```diff
@@ -147,16 +131,8 @@ describe('Hits Resolver', () => {
             Object {
               "aggs": Object {},
               "from": 10,
-              "post_filter": Object {
-                "bool": Object {
-                  "must": Array [],
-                },
-              },
-              "query": Object {
-                "bool": Object {
-                  "filter": Array [],
-                },
-              },
+              "post_filter": Object {},
```

Comment:
If `post_filter` & `query` is an empty object, omit these from the query, otherwise its an invalid ES query. 

      "bool":  {
        "must":  []
      }